include/CL/cl_exp_defined_builtin_kernels.h

```diff
@@ -261,4 +263,79 @@ typedef struct _cl_dbk_attributes_onnx_inference_exp
   const char **initializer_data;
 } cl_dbk_attributes_onnx_inference_exp;
 
+/**
+ * Collection of different color spaces that can be used
+ * to populate entries in pocl_image_attr_t.
+ */
+enum pocl_color_space_e
+{
+  POCL_COLOR_SPACE_BT601_525,
+  POCL_COLOR_SPACE_BT709,
+  POCL_COLOR_SPACE_DEFAULT = POCL_COLOR_SPACE_BT709,
+};
+
+/**
+ * Collection of different image formats that can be used
+ * to populate entries in pocl_image_attr_t.
+ * @todo describe each type
+ */
+enum pocl_format_e
+{
+  POCL_DF_IMAGE_RGB,
+  POCL_DF_IMAGE_NV12,
+  POCL_DF_IMAGE_NV21,
+  POCL_DF_IMAGE_IYUV,
+};
+
+/**
+ * Collection of image channel ranges that can be used
+ * to populate entries in pocl_image_attr_t.
+ */
```

Comment:
Could this be expanded further? Not sure what these options do.

Reply:
A bit of context for the design choices I made with the color convert kernel is that I tried to make it very similar to OpenVX and could easily be dropped into what has already been done with PoVX. See https://registry.khronos.org/OpenVX/specs/1.3/html/OpenVX_Specification_1_3.html#group_vision_function_colorconvert , for a more detailed explanation of the channel ranges.

Reply:
Could you add this to the comment? The link you provided probably suffices.

Reply:
Yes, already done. A commit is coming and i'm just resolving these conversations as checking off todo boxes.